src/transform3d.rs

```diff
@@ -633,6 +633,23 @@ where T: Copy + fmt::Debug +
     }
 }
 
+impl<T: Copy, Src, Dst> Into<[T; 16]> for TypedTransform3D<T, Src, Dst> {
```

Comment:
In practice, most 2d rendering libraries (cairo, skia, moz2d, etc.) store 2d transforms the way we do so using arrays of 6 elements is is what's most consistent in a useful way. 
If we added the implicit components we could serialize but not deserialize since we would not know what to do with the extra values if they are not zero.